### PR TITLE
feature(structure):RR-0010: Create apps based on structure

### PR DESCRIPTION
**Pull request by Jann337**
https://trello.com/c/nAVXCqiZ/12-featrr-0010-create-empty-apps

### CODE REVIEW

File: front_office/scroll_down_page/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (EimantasBlazevicius) [inline front_office/scroll_down_page/admin.py]: delete this app, this functionality is part of base setup and we will not need an app for that. 
Just delete this scroll_down_page folder.

File: back_office/add_products/apps.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AddProductsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'back_office.add_products'
```
Comment (EimantasBlazevicius) [inline back_office/add_products/apps.py]: App should be called just products, as there will be A LOT more functionality in here than just add products, we will delete, update, edit and do all the product related things here.

I suggest delete this folder and create new app named products.

**Review by EimantasBlazevicius — CHANGES_REQUESTED** [review]

File: front_office/scroll_down_page/admin.py (hunk 1 above)
Comment (Jann337) [reply front_office/scroll_down_page/admin.py]: will do.

File: back_office/add_products/apps.py (hunk 2 above)
Comment (Jann337) [reply back_office/add_products/apps.py]: wil change to products then.

**Review by EimantasBlazevicius — APPROVED** [review]

**Review by v1ljar — APPROVED** [review]

**Pull request merged**